README.md

```diff
@@ -1,104 +1,159 @@
 
+### Included modules
 
-# What's on board
+The following Lua modules are included in the build:
 
-- [avro-schema](https://github.com/tarantool/avro-schema): Apache Avro scheme for your data
-- [expirationd](https://github.com/tarantool/expirationd): Automatically delete tuples based on expiration time
-- [queue](https://github.com/tarantool/queue): Priority queues with TTL and confirmations
-- [connpool](https://github.com/tarantool/connpool): Keep a pool of connections to other Tarantool instances
-- [vshard](https://github.com/tarantool/vshard): Automatically distribute data across multiple instances
-- [http](https://github.com/tarantool/http): Embedded HTTP server with flask-style routing support
-- [curl](https://github.com/tarantool/curl): HTTP client based on libcurl
-- [pg](https://github.com/tarantool/pg): Query PostgreSQL right from Tarantool
-- [mysql](https://github.com/tarantool/mysql): Query MySql right from Tarantool
-- [memcached](https://github.com/tarantool/memcached): Access Tarantool as if it was a Memcached instance
-- [metrics](https://github.com/tarantool/metrics): Metric collection library for Tarantool
-- [prometheus](https://github.com/tarantool/prometheus): Instrument code and export metrics to Prometheus monitoring
-- [mqtt](https://github.com/tarantool/mqtt): Client for MQTT message brokers
-- [gis](https://github.com/tarantool/gis): store and query geospatial data
-- [gperftools](https://github.com/tarantool/gperftools): collect CPU profile to find bottlenecks in your code
+- [avro-schema](https://github.com/tarantool/avro-schema): Apache Avro scheme for your data.
+- [connpool](https://github.com/tarantool/connpool): Keep a pool of connections to other Tarantool instances.
+- [curl](https://github.com/tarantool/curl): HTTP client based on libcurl.
+- [expirationd](https://github.com/tarantool/expirationd): Automatically delete tuples based on expiration time.
+- [gis](https://github.com/tarantool/gis): Store and query geospatial data.
+- [gperftools](https://github.com/tarantool/gperftools): Collect CPU profile to find bottlenecks in your code.
+- [http](https://github.com/tarantool/http): Embedded HTTP server with flask-style routing support.
+- [memcached](https://github.com/tarantool/memcached): Access Tarantool as if it was a Memcached instance.
+- [metrics](https://github.com/tarantool/metrics): Metric collection library for Tarantool.
+- [mqtt](https://github.com/tarantool/mqtt): Client for MQTT message brokers.
+- [mysql](https://github.com/tarantool/mysql): Query MySQL right from Tarantool.
+- [pg](https://github.com/tarantool/pg): Query PostgreSQL right from Tarantool.
+- [prometheus](https://github.com/tarantool/prometheus): Instrument code and export metrics to Prometheus monitoring.
+- [queue](https://github.com/tarantool/queue): Priority queues with TTL and confirmations.
+- [vshard](https://github.com/tarantool/vshard): Automatically distribute data across multiple instances.
```

Comment:
I'd propose a preferential change. I'd consider making the list items grammatically parallel, i.e. all noun phrases or all imperative sentences.
Example:
```
mysql: A tool to query MySQL right from Tarantool.
queue: Priority queues with TTL and confirmation.
```
or:
```
mysql: Query MySQL right from Tarantool.
queue: Create priority queues with TTL and confirmation.
```